File: src/main/java/com/my/list/domain/Part.java
package com.my.list.domain;

import lombok.Data;

@Data
public class Part {

    private Long id;
    private Long parentId;
    private Long contentId;
    private Integer contentOrder;

    @Override
    public String toString() {
        return "Part{" +
            "id=" + id +
            ", parentId=" + parentId +
            ", contentId=" + contentId +
            ", contentOrder=" + contentOrder +
            '}';
    }
}

File: src/main/java/com/my/list/domain/Part.kt
package com.my.list.domain

class Part(
    var id: Long? = null,
    var parentId: Long? = null,
    var contentId: Long? = null,
    var contentOrder: Int? = null
) {
    override fun toString(): String {
        return "Text[$id,$parentId,$contentId,$contentOrder]"
    }
}
